templates/cluster_resources.j2

```diff
@@ -77,6 +77,22 @@ primitive rsc_ip_{{ sid }}_ERS{{ ers_instance }} IPaddr2 \
 
 {%- endif %}
 
+{%- if data.monitoring_enabled %}
+
+primitive rsc_exporter_{{ sid }}_ASCS{{ ascs_instance }} systemd:prometheus-sap_host_exporter@{{ sid }}_ASCS{{ ascs_instance }} \
+    op start interval=0 timeout=60 \
```

Comment:
@stefanotorresi As a comment, `crmsh` complains that the `start/stop` timeouts are smaller than the recommended 100 value. We should change it in the ASCS and ERS exporter

```
WARNING: rsc_exporter_HA1_ASCS00: specified timeout 60 for start is smaller than the advised 100
WARNING: rsc_exporter_HA1_ASCS00: specified timeout 60 for stop is smaller than the advised 100
WARNING: rsc_exporter_HA1_ERS10: specified timeout 60 for start is smaller than the advised 100
WARNING: rsc_exporter_HA1_ERS10: specified timeout 60 for stop is smaller than the advised 100
```